### ISSUE TITLE
Factorio Mod Page and another incompatibility (External Rocket Parts)

### ISSUE THREAD

**Issue by KingDeDeDe11** [nicholasgower/planet-muluna#20]
**Factorio & Muluna version number**
Factorio 2.0.28 (build 810088 expansion, win64)
Muluna 1.9.6
External Rocket Parts 0.0.3

On the Factorio Mod Page under information it lists "any mod that removes vanilla terrain features, such as Alien Biomes, is likely to crash on startup" but then below that 
"Known to be compatible with:

Alien Biomes"

This likely should be incompatible rather than compatible.

Another note is that this mod is also currently incompatible with External Rocket Parts. The rocket silo is being forced back to internal rocket parts, preventing the external ones from working.

**Comment by KingDeDeDe11** [comment]
or does this mean that it is compatible. I may have misread it. Oh well, feel free to ignore this if that's the case.

**Comment by nicholasgower** [comment]
Yeah, you did misread my description, but I appreciate you reaching out. It's possible there's a better way of describing that fact. As for External Rocket Parts, I'll add an incompatibility flag to prevent others from trying to use this mod with it in the next release. I'm not familiar with this mod, so adding compatibility for it is not a priority. The rocket silo code that changes rocket part recipes is lifted from Maraxsis, so this incompatibility is likely also present in Maraxsis. Feel free to test Maraxsis, and if the incompatibility is also present in that mod, reach out to them about this incompatibility. If NotNotMelon decides to issue a fix, it will likely be possible for me to add that fix to my mod as well.

**Comment by KingDeDeDe11** [comment]
Interestingly we have Maraxsis installed and it allows us to insert the rocket parts as expected. Again, not a huge issue as external rocket parts is a small mod from what I can see, but I figured I'd report it when I found it.

**Comment by nicholasgower** [comment]
Now that Muluna is in a more mature state, I decided to look into adding compatibility with External Rocket parts, thinking it would be a simple job. Looking at the source code for External rocket parts, I see that they changed the recipe name used for rocket parts, which makes it a lot more difficult to add compatibility for it without explicitly looking for this mod. If there was a version of this mod that was better designed that needed a compatch, I'd happily make it, but not this particular mod.

**Comment by Tramdytojas** [comment]
by using this mod (muluna,moon of nauvis) i cant seem to trigger research for gleba disscovery, i already build full operating base on Muluna. can research Fulgora and Vulcanus, but i cant research Gleba no matter what i do
